src/content/structured/components/select/code.mdx

```diff
@@ -2015,27 +2016,38 @@ return (
   },
 ];
 
+export const ReadonlyMulti = () => {
+  useEffect(() => {
+    document
+      .querySelector("#readonly-multi-select")
```

Comment:
better to use `useRef` rather than doing document.querySelector? more react-like that way